app/models/shelter.rb

```diff
@@ -35,4 +36,8 @@ def shelter_pets_filtered_by_age(age_filter)
   def self.order_by_alpha_desc
     self.find_by_sql('SELECT * FROM shelters ORDER BY name DESC')
   end
+
+  def self.pending_app_shelters
+    Shelter.joins(:pets).joins(:applications).where(Application.arel_table[:status].matches("Pending")).distinct
```

Comment:
Great, we'll review this, I have a simpler way of doing that 'cause I dunno what arels are